src/plugins/discover/public/application/components/sidebar/discover_field.scss

```diff
@@ -2,3 +2,7 @@
   min-width: 260px;
   max-width: 300px;
 }
+
+.dscSidebarItem__fieldPopoverPanel__unmappedSwitch {
+  margin-top: $euiSizeS;
+}
```

Comment:
I don't think we're using this class anywhere?

Reply:
Good catch!